spec/source_finder/source_file_globber_spec.rb

```diff
@@ -34,10 +35,10 @@
   describe '#source_and_doc_files_glob' do
     context 'with everything unconfigured' do
       expected_glob =
-        '{Rakefile,Dockerfile,{*,.*}.{md,rb,rake,gemspec,swift,cpp,c,java,' \
-        'py,clj,cljs,scala,js,yml,sh,json},{src,app,config,db,lib,test,' \
-        'spec,feature}/**/{*,.*}.{md,rb,rake,gemspec,swift,cpp,c,java,' \
-        'py,clj,cljs,scala,js,yml,sh,json}}'
+        '{Dockerfile,Rakefile,{*,.*}.{c,clj,cljs,cpp,gemspec,java,js,json,' \
+        'md,py,rake,rb,scala,sh,swift,yml},{app,config,db,feature,lib,spec,' \
+        'src,test,www}/**/{*,.*}.{c,clj,cljs,cpp,gemspec,java,js,json,md,py,' \
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.